bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -279,13 +279,22 @@ JsErrorCode CreateHostContext(JsRuntimeHandle runtime, std::vector<std::wstring>
     JsValueRef globalObject = JS_INVALID_REFERENCE;
     IfFailRet(JsGetGlobalObject(&globalObject));
 
+    JsValueRef consoleObject = JS_INVALID_REFERENCE;
+    IfFailRet(handler->GetConsoleObject(&consoleObject));
+
     // Get the name of the property ("host") that we're going to set on the global object.
     JsPropertyIdRef hostPropertyId = JS_INVALID_REFERENCE;
     IfFailRet(JsGetPropertyIdFromName(L"host", &hostPropertyId));
 
+    JsPropertyIdRef consolePropertyId = JS_INVALID_REFERENCE;
+    IfFailRet(JsGetPropertyIdFromName(L"console", &consolePropertyId));
```

Comment:
Can you add some usage examples to the sample JS file?

Reply:
Sure will do.